.env.example

```diff
@@ -15,6 +15,7 @@ MYUSA_SECRET: consumer_secret_key
 # DEFAULT_URL_PORT
 # DEFAULT_URL_SCHEME
 # FLAG_ITEM_COMMENT_UI
+# FORCE_HTTPS=true
```

Comment:
All the lines in `app/assets/javascripts/application.js` are commented, too, but things aren't always what they appear in rails land ;)